keyboards/miuni32/keymaps/cassdelacruzmunoz/keymap.c

```diff
@@ -0,0 +1,141 @@
+  ),
+  /* Level 5: Commands Layer
+     * ,---------------------------------------------------------------------------------------.
+     * |  CUT  |  PSCR |  BRIU |  PWR  |  TRNS |  TRNS |  TRNS |  TRNS |  TRNS |  TRNS |  TRNS |
+     * |---------------------------------------------------------------------------------------|
+     * |  COPY |  FIND |  BRID |  SLEP |  TRNS |  TRNS |  TRNS |  TRNS |  TRNS |  TRNS | TG(5) |
+     * |---------------------------------------------------------------------------------------|
+     * |  PSTE |  TRNS |  TRNS |  WAKE |      TRNS     |  TRNS |  TRNS |  TRNS |  TRNS |  TRNS |
+     * |---------------------------------------------------------------------------------------|
+     */
+  [5] = LAYOUT(
+    KC_CUT,  KC_PSCR, KC_BRIU, KC_PWR, _______, _______, _______, _______, _______, _______, _______,
+    KC_COPY, KC_FIND, KC_BRID, KC_SLEP, _______, _______, _______, _______, _______, _______, TG(5),
+    KC_PSTE, _______, _______, KC_WAKE,          _______, _______, _______, _______, _______, _______
+  )
+};
+
+// Light LEDs 0 through 16 in white when keyboard layer 0 is active
+const rgblight_segment_t PROGMEM my_layer0_layer[] = RGBLIGHT_LAYER_SEGMENTS(
+    {0, 17, HSV_WHITE}
+);
+// Light LEDs 0 through 16 in blue when keyboard layer 1 is active
+const rgblight_segment_t PROGMEM my_layer1_layer[] = RGBLIGHT_LAYER_SEGMENTS(
+    {0, 17, HSV_BLUE}
+);
+// Light LEDs 0 through 16 in green when keyboard layer 2 is active
+const rgblight_segment_t PROGMEM my_layer2_layer[] = RGBLIGHT_LAYER_SEGMENTS(
+    {0, 17, HSV_GREEN}
+);
+// Light LEDs 0 through 16 in yellow when keyboard layer 3 is active
+const rgblight_segment_t PROGMEM my_layer3_layer[] = RGBLIGHT_LAYER_SEGMENTS(
+    {0, 17, HSV_YELLOW}
+);
+// Light LEDs 0 through 16 in red when keyboard layer 4 is active
+const rgblight_segment_t PROGMEM my_layer4_layer[] = RGBLIGHT_LAYER_SEGMENTS(
+    {0, 17, HSV_RED}
+);
+// Light LEDs 0 through 16 in cyan when keyboard layer 5 is active
+const rgblight_segment_t PROGMEM my_layer5_layer[] = RGBLIGHT_LAYER_SEGMENTS(
+    {0, 17, HSV_CYAN}
+);
+
+// Now define the array of layers. Later layers take precedence
+const rgblight_segment_t* const PROGMEM my_rgb_layers[] = RGBLIGHT_LAYERS_LIST(
+    my_layer0_layer,
+    my_layer1_layer,
+    my_layer2_layer,
+    my_layer3_layer,
+    my_layer4_layer,
+    my_layer5_layer
+);
+
+void keyboard_post_init_user(void) {
+    // Enable the LED layers
+    rgblight_layers = my_rgb_layers;
+}
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    rgblight_set_layer_state(0, layer_state_cmp(state, 0));
+    rgblight_set_layer_state(1, layer_state_cmp(state, 1));
+    rgblight_set_layer_state(2, layer_state_cmp(state, 2));
+    rgblight_set_layer_state(3, layer_state_cmp(state, 3));
+    rgblight_set_layer_state(4, layer_state_cmp(state, 4));
+    rgblight_set_layer_state(5, layer_state_cmp(state, 5));
```

Comment:
My for loop logic is iffy, but you should be able to do something like this: 

```suggestion
    for (uint8_t i = 0, i =< 5, i++) {
        rgblight_set_layer_state(i, layer_state_cmp(state, i));
    }
```

Reply:
Commit 220438f implements the loop.
Turns out that a regular int is needed, not the uint8_t.